docs/en/flink/configuration/sink-plugins/Kafka.md

```diff
@@ -29,14 +30,17 @@ In addition to the above mandatory parameters that must be specified by the `Kaf
 
 The way to specify the parameter is to add the prefix `producer.` to the original parameter name. For example, the way to specify `request.timeout.ms` is: `producer.request.timeout.ms = 60000` . If these non-essential parameters are not specified, they will use the default values given in the official Kafka documentation.
 
+### semantic [string]
+Semantics that can be chosen. exactly_once/at_least_once/none, default is at_least_once
```

Comment:
The introduction of semantic parameters can be a little more detailed, like `equal with kafka parameters`?